applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -154,6 +154,68 @@ config BLE_ADV_TX_POWER_DBM
 	default -20 if BLE_ADV_TX_POWER_NEG_20DBM
 	default -40 if BLE_ADV_TX_POWER_NEG_40DBM
 
+config NRF_21540_ACTIVE
+	bool "Use front end module nRF 21540"
+	default n
+	help
+	  The front end module can help boost the TX power as high as 20 dBm.
+	  Usually set through buildprog, if not set there, remember to add
+	  -DSHIELD=nrf21540_ek_fwd when building app core
```

Comment:
What happens if `-DSHIELD=nrf21540_ek_fwd` is set but NRF_21540_ACTIVE=n?

Reply:
Will check if Kconfig know if shield included or not

Reply:
Could not find it, so removed the prompt so that it doeesn't show in menuconfig. This will then have to be set through -DCONFIG_NRF_21540_ACTIVE when building, either through west or buildprog.